allennlp/data/tokenizers/word_splitter.py

```diff
@@ -33,8 +33,7 @@ def from_params(cls, params: Params) -> 'WordSplitter':
         choice = params.pop_choice('type', cls.list_available(), default_to_first_choice=True)
         # None of the word splitters take parameters, so we just make sure the parameters are empty
         # here.
-        params.assert_empty('WordSplitter')
-        return cls.by_name(choice)()
+        return cls.by_name(choice).from_params(params)
```

Comment:
fix comment, we're not making sure the parameters are empty

Reply:
Done.